projects/plugins/jetpack/class.jetpack.php

```diff
@@ -1493,7 +1498,8 @@ public static function is_multi_network() {
 	public static function is_single_user_site() {
 		global $wpdb;
 
-		if ( false === ( $some_users = get_transient( 'jetpack_is_single_user' ) ) ) {
+		$some_users = get_transient( 'jetpack_is_single_user' );
+		if ( false === ( $some_users ) ) {
```

Comment:
Do we need the parenthesis here?

```suggestion
		if ( false === $some_users ) {
```